shell/platform/fuchsia/flutter/vulkan_surface.cc

```diff
@@ -370,89 +388,21 @@ bool VulkanSurface::SetupSkiaSurface(sk_sp<GrDirectContext> context,
   return true;
 }
 
-bool VulkanSurface::PushSessionImageSetupOps(scenic::Session* session) {
-  FML_DCHECK(scenic_memory_ != nullptr);
-
-  if (sk_surface_ == nullptr) {
```

Comment:
I noticed that this check was removed. I assume that's intentional?

Reply:
This check doesn't seem to be needed anymore since the only place where `PushSessionImageSetupOps` is in the constructor where `sk_surface_` being `nullptr` would return before this can execute.